ConsoleGame.java

```diff
@@ -0,0 +1,46 @@
+public class ConsoleGame {
+    private final Game game;
+
+    public ConsoleGame(Game game) {
+        this.game = game;
```

Comment:
As we discussed, the ConsoleGame is concerned with IO so should do display behaviour, but also input retrieval behaviour.